src/cmd/build.js

```diff
@@ -131,7 +131,8 @@ async function defaultPackageCreator({
   await eventToPromise(stream, 'close');
 
   if (showReadyMessage) {
-    log.info(`Your web extension is ready: ${extensionPath}`);
```

Comment:
I meant, in the output of `web-ext build`, like this (I'm not sure what would the good wording for it is tho):

```
Building web extension from /tmp/test
Your web extension is ready: /tmp/test/web-ext-artifacts/test-1.0.zip
NOTE: the generated zip file may not work on Firefox without signing it.
```

about the documentation, I'll change it to suggest signing first.